wagtail/admin/templates/wagtailadmin/edit_handlers/tabbed_interface.html

```diff
@@ -1,20 +1,21 @@
 {% load wagtailadmin_tags i18n %}
-<ul class="tab-nav merged" role="tablist" data-current-tab="{{ self.children.0.heading|cautious_slugify }}">
-    {% for child in self.children %}
-        <li class="{{ child.classes|join:" " }} {% if forloop.first %}active{% endif %}" role="tab" aria-controls="tab-{{ child.heading|cautious_slugify }}">
-            <a href="#tab-{{ child.heading|cautious_slugify }}" class="{% if forloop.first %}active{% endif %}" data-tab="{{ child.heading|cautious_slugify }}">{{ child.heading }}</a>
-        </li>
-    {% endfor %}
-
+<div class="tab-nav merged">
+    <ul data-tab-nav role="tablist" data-current-tab="{{ self.children.0.heading|cautious_slugify }}">
+        {% for child in self.children %}
+            <li class="{{ child.classes|join:" " }} {% if forloop.first %}active{% endif %}" role="tab" aria-controls="tab-{{ child.heading|cautious_slugify }}">
+                <a href="#tab-{{ child.heading|cautious_slugify }}" class="{% if forloop.first %}active{% endif %}" data-tab="{{ child.heading|cautious_slugify }}">{{ child.heading }}</a>
+            </li>
+        {% endfor %}
+    </ul>
     {% if self.show_comments_toggle %}
-        <li class="right wide">
+        <span class="right wide">
             <div class="comments-controls">
                 <label class="comments-toggle" aria-label="{% trans 'Show comments' %}">
                     <input type="checkbox" checked>
                     {% icon name="comment" class_name="comments-toggle__icon" %}
                     <div class="comments-toggle__count">{# populated in page-editor.js #}</div>
                 </label>
-
+        
```

Comment:
Trailing space, you can set up your editor to remove this automatically

Reply:
Thanks, should be removed now! Thought I had got that set up, but apparently only on certain file types - hopefully my editor will catch it now.